pkg/reconciler/cache/replication/replication_reconcile.go

```diff
@@ -82,22 +82,22 @@ func (c *controller) reconcile(ctx context.Context, gvrKey string) error {
 
 // reconcileObject makes sure that the object under the given key from the local shard is replicated to the cache server.
 // the replication function handles the following cases:
-//  1. creation of the object in the cache server when the cached object is not found by retriveLocalObject
```

Comment:
nit: not including unrelated stuff helps to review :)

Reply:
same as above